analyzer/api.go

```diff
@@ -162,43 +162,43 @@ const (
 func (e *Event) String() string {
 	switch *e {
 	case EventStakingTransfer:
-		return "Transfer"
+		return "staking.transfer"
 	case EventStakingBurn:
-		return "Burn"
+		return "staking.burn"
 	case EventStakingAddEscrow:
-		return "AddEscrow"
+		return "staking.escrow.add"
 	case EventStakingTakeEscrow:
-		return "TakeEscrow"
+		return "staking.escrow.take"
 	case EventStakingDebondingStart:
-		return "DebondingStart"
+		return "staking.escrow.debonding_start"
```

Comment:
The current naming style (with dots) is probably the most readable. A downside is that some openapi autogenerators might be unhappy with dots in enum values. I could go with a double underscore (`staking__escrow__debonding_start`) or similar if anybody has concerns/suggestions. 

I'd say it's fine as-is though. The openapi spec doesn't limit the character set, and when I saw somebody complaining online about a generator not accepting a special char (`%` in their case), it was in the context of a github issue, and it was considered a genuine bug and fixed.